democracy/models/comment.py

```diff
@@ -24,11 +24,10 @@ class BaseComment(BaseModel):
         default=0,
         editable=False
     )
-    n_legacy_votes = models.IntegerField(
-        verbose_name=_('legacy vote count'),
-        help_text=_('number of votes imported from legacy system'),
-        default=0,
-        editable=False
+    n_unregistered_votes = models.IntegerField(
+        verbose_name=_('unregistered vote count'),
+        help_text=_('number of unregistered votes'),
+        default=0
```

Comment:
Why was the editable flag dropped? I don't think this should be editable by humans. ([The `editable` flag does not make a field read-only.](https://docs.djangoproject.com/en/1.10/ref/models/fields/#editable))